test/contracts/MarginBase.t.sol

```diff
@@ -1134,20 +1018,29 @@ contract MarginBaseTest is DSTest {
             abi.encode(Position(0, 0, 1 ether, 1 ether, 0)) // size = 0
```

Comment:
what's the problem with mocking this to return an active position size again?

Reply:
When calling `IFuturesMarket.positions()`, our contract learns whether or not a position is `active`. 

Two scenarios are possible:
(1) An external call to futures contracts has altered the position so that the `size` becomes **zero** (i.e. closing the position)
(2) The position was liquidated, thus `size` is **zero**

Notice in both, `size` is **zero**...

While testing, we mock `IFuturesMarket.positions()` to return `0`. However, in our contract, we make two calls to that function to determine whether the position was liquidated (first check) and finally whether our actions closed the position (second check).

Since both checks rely on the same function call and occur in the same tx, andddd they check for the same thing (is `size` zero), we can thus never have the first check return a *non-zero* value (the position wasn't liquidated) and the second check return *zero*.